webapp/src/main/java/com/box/l10n/mojito/service/smartling/ThirdPartyTextUnitDTONativeObjectMapper.java

```diff
@@ -0,0 +1,19 @@
+package com.box.l10n.mojito.service.smartling;
+
+import com.github.pnowy.nc.core.CriteriaResult;
+import com.github.pnowy.nc.core.mappers.NativeObjectMapper;
+
+public class ThirdPartyTextUnitDTONativeObjectMapper implements NativeObjectMapper<ThirdPartyTextUnitDTO> {
```

Comment:
we're gonna use Native SQL queries?

Reply:
similar set up to TextUnitDTONativeObjectMapper, uses NativeCriteria

Reply:
not sure why NativeCriteria is needed here? right now it is only used for searching text units which require complicated queries. 

For the rest we use standard queries, even when using DTOs (`AssetMappingDTO`, `TextUnitIdMd5DTO`, etc)

Reply:
Got it, I will remove this file since NativeCriteria is overkill